docs/pages/components/cldogimage/configuration.mdx

```diff
@@ -38,6 +39,30 @@ The `summary_large_image` Twitter card type requires the inclusion of a `twitter
 
 To exclude this tag to manage it on your own, use the `excludeTags` prop:
 
-```
+```jsx
 excludeTags={['twitter:title']}
+```
+
+## Keys
+
+By default, CldOgImage includes static keys on each meta tag rendered to help avoid duplication in the DOM.
+
+To allow customization and control in the event you want to minimize duplication with existing meta tags, you can specify custom keys for each tag.
+
+For instance, by default `og:image` tag will effectively render with:
+
+```jsx
+<meta key="og-image" property="og:image" content="..." />
+```
+
+Where if you pass in the `keys` prop with the following:
+
+```jsx
+keys={{'og:image': 'my-og-image'}}
+```
+
+The tag will render with:
+
+```jsx
+<meta key="og-image" property="my-og-image" content="..." />
```

Comment:
Should this key by `my-og-image` instead of `og-image`?

Reply:
And I think the `property` property should have stayed the same as `og-image`

Reply:
oops yeah thanks good catch